<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindMobileWidget</class>
 <widget class="QWidget" name="BindMobileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>254</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>454</width>
    <height>254</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>454</width>
    <height>254</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModifyCommonWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#BindMobileWidget{background-color: rgb(255, 255, 255);border-radius:8px;}</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>10</y>
     <width>32</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>100</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>手机绑定</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetMobile" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>454</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>454</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>454</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>32</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>45</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string>手机号</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMobile">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>20</y>
      <width>268</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>268</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>268</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
    </property>
    <property name="placeholderText">
     <string>请输入您的手机号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMobileCode">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>80</y>
      <width>268</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>268</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>268</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
    </property>
    <property name="placeholderText">
     <string>请输入验证码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>90</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>45</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string>验证码</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>261</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string>请输入正确的验证码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>141</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string>请输入11位正确的手机号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>150</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>72</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>72</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>150</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>72</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>72</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#FFFFFF;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE;}
QPushButton:hover,pressed{background-color:#F4F4F4;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCodeButton">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>92</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#FFFFFF;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#1E8BED;}
QPushButton:hover,pressed{background-color:#FFFFFF;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:rgba(64, 156, 240, 1);}
QPushButton:disabled{ background-color:#FFFFFF;color:#666666};</string>
    </property>
    <property name="text">
     <string>发送验证码</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhone">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>268</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>268</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>268</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: #333333;
border-bottom:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
